DependencyInjection/CmfCreateExtension.php

```diff
@@ -86,20 +86,25 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadSecurity($config['security'], $loader, $container);
 
-        $hasMapper = false;
         if ($config['persistence']['phpcr']['enabled']) {
             $this->loadPhpcr($config['persistence']['phpcr'], $loader, $container);
-            $hasMapper = true;
         } else {
             // TODO: we should leverage the mediabundle here and not depend on phpcr
             $container->setParameter($this->getAlias() . '.image_enabled', false);
         }
+        if ($config['persistence']['orm']['enabled']) {
```

Comment:
it's better to use `if ($this->isConfigEnabled($config['persistence']['orm'])) {`